diff --git a/utils/bazel/llvm-project-overlay/clang/BUILD.bazel b/utils/bazel/llvm-project-overlay/clang/BUILD.bazel
index 6a61b99f9..5cce43bb1 100644
--- a/utils/bazel/llvm-project-overlay/clang/BUILD.bazel
+++ b/utils/bazel/llvm-project-overlay/clang/BUILD.bazel
@@ -2,6 +2,7 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

+load("@io_kythe//tools:build_rules/support.bzl", glob = "allow_empty_glob")
 load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
 load("//:workspace_root.bzl", "workspace_root")
 load("//llvm:tblgen.bzl", "gentbl")
@@ -18,7 +19,7 @@

 package(
     default_visibility = ["//visibility:public"],
-    features = ["layering_check"],
+    features = ["-layering_check"],
 )

 licenses(["notice"])
diff --git a/utils/bazel/llvm-project-overlay/llvm/BUILD.bazel b/utils/bazel/llvm-project-overlay/llvm/BUILD.bazel
index 93e905260..2c3cedcd9 100644
--- a/utils/bazel/llvm-project-overlay/llvm/BUILD.bazel
+++ b/utils/bazel/llvm-project-overlay/llvm/BUILD.bazel
@@ -2,6 +2,7 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

+load("@io_kythe//tools:build_rules/support.bzl", glob = "allow_empty_glob")
 load("@bazel_skylib//rules:common_settings.bzl", "string_flag")
 load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
 load(":tblgen.bzl", "gentbl")
@@ -12,7 +13,7 @@

 package(
     default_visibility = ["//visibility:public"],
-    features = ["layering_check"],
+    features = ["-layering_check"],
 )

 licenses(["notice"])
